site/en/reference/sys_config/configure_knowhere.md

```diff
@@ -0,0 +1,34 @@
+---
+id: configure_knowhere.md
+related_key: configure
+summary: Learn how to configure common parameters of Milvus.
+---
+
+# Common Configurations
+
+This topic introduces the knowhere-related configurations of Milvus.
+
+Knowhere is the search engine of Milvus.
```

Comment:
[Knowhere](https://github.com/milvus-io/milvus/blob/master/docs/design_docs/knowhere_design.md) is the search engine of Milvus.